doc/source/user_guide/hpc_slurm.rst

```diff
@@ -0,0 +1,611 @@
+
+.. _ref_hpc_slurm:
+
+=================
+SLURM HPC cluster
+=================
+
+Introduction
+============
+
+High-Performance Computing (HPC) clusters are powerful systems designed to handle
```

Comment:
```suggestion
High performance computing (HPC) clusters are powerful systems designed to handle
```
Google dev style guide says not to hyphenate high performance computing. I assume that you are defining HPC here because it will be the first reference that the reader sees?